spec/features/users/user_index_spec.rb

```diff
@@ -1,25 +1,23 @@
 include Warden::Test::Helpers
 Warden.test_mode!
 
-# Feature: User index page
-#   As a user
-#   I want to see a list of users
-#   So I can see who has registered
-feature 'User index page', :devise do
-
+feature 'User index page', %{
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.